format.ts

```diff
@@ -4,7 +4,7 @@ import { parse } from "https://deno.land/x/flags/mod.ts";
 const { exit, args, run } = Deno;
 
 async function main(opts) {
-  const args = ["deno", "--allow-run", "--fmt", "--ignore", "lib"];
+  const args = ["deno", "fmt", "--", "--ignore", "lib"];
```

Comment:
Is the -- needed? (Is this a deno bug?)

Reply:
`--ignore` is argument that is parsed inside `deno_std/prettier/main.ts` - it's not handled by Deno CLI itself, to `--` informs clap's parser that the rest should be forwarded to the script without being parsed